wsgi/odyssy/events/views.py

```diff
@@ -0,0 +1,24 @@
+from django.shortcuts import get_object_or_404
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+from django.shortcuts import render
+from .models import Event
+
+
+def event(request):
```

Comment:
Why to use an extra parameter here. We could obtain it from `request.GET.get()`.